### PR TITLE
FIX: brain RGBA screenshot

### PR DESCRIPTION
**Pull request by GuillaumeFavelier**
This PR fixes https://github.com/mne-tools/mne-python/issues/9562

I was able to reproduce it locally, can you try it now @hoechenberger ?

### CODE REVIEW

**Comment by larsoner** [comment]
Can you add a test and update `latest.inc`?

**Comment by hoechenberger** [comment]
@GuillaumeFavelier I can confirm that this fixes the issue for me. Thanks a bunch!

**Review by hoechenberger — APPROVED** [review]

File: doc/changes/latest.inc [hunk 1]
```diff
@@ -127,6 +127,8 @@ Bugs
 
 - Fix bug in :func:`mne.concatenate_epochs` when concatenating :class:`mne.Epochs` objects with 0 events (:gh:`9535` by `Marijn van Vliet`_)
 
+- Fix bug in :func:`mne.viz.Brain.screenshot` where the RGBA mode was not supported (:gh:`9561` by `Guillaume Favelier`_).
```
Comment (larsoner) [inline doc/changes/latest.inc:130]: Wrong number, this was 9564

Comment (larsoner) [reply doc/changes/latest.inc:130]: Fixed the number in #9565

**Pull request merged**